packages/store-fs/src/sqlite.ts

```diff
@@ -44,10 +44,13 @@ export class SQLDatabaseBackend implements DatabaseBackend {
 
   static attachmentURLProtocol = "x-orbit-sqlattachment";
 
-  constructor(path: string) {
+  constructor(path: string, options?: { enableDebugLogs?: boolean }) {
```

Comment:
these logs were becoming quite noisy in the unit tests so I created a flag to disable them. Happy to revert these changes and just stub out `console.log` in my tests if you would prefer.

The more SOLID approach would to pass in a `Logger` interface, but that felt a bit heavy for this case.